src/Npgsql/TypeHandlers/CompositeHandlers/CompositeMemberHandlerOfClass.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Diagnostics;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Threading.Tasks;
+using Npgsql.PostgresTypes;
+using Npgsql.TypeHandling;
+
+namespace Npgsql.TypeHandlers.CompositeHandlers
+{
+    sealed class CompositeMemberHandlerOfClass<TComposite, TMember> : CompositeMemberHandler<TComposite>
```

Comment:
Proposal: rename to CompositeClassMemberHandler and CompositeStructMemberHandler.